blocks/full-author-bio-block/features/full-author-bio/default.test.jsx

```diff
@@ -199,7 +200,7 @@ describe('the full author bio block', () => {
       it('should have an instagram url', () => {
```

Comment:
we can probably delete this test. it's basically just testing the mock as far as I can see.